src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,21 @@
+import Loading from './Loading'
```

Comment:
This isn't being used anywhere, on a slow connection I just have a blank screen.